src/signing.js

```diff
@@ -7,6 +7,8 @@
 
 var actualMethods = {};
 
+export var fastSigning;
```

Comment:
Good to know about the export change...  I guess at some point it changed in the standard from how babel behaved.  The behavior is not very clear to me, even looking at the discussion on the final syntax @ http://www.2ality.com/2014/09/es6-modules-final.html